### DISCUSSION TITLE
Running Homebride Ring on Node.js v17.9.1

### DISCUSSION THREAD

**Discussion by AndySound** [dgreif/ring#1370] [Q&A] [answered]
I'm running Homebride on MacOS 10.13.6 High Siera and getting the error to update Node.js from v17.9.1 to Node.js v18, but v18 is for Catalina 10.15. I have the Ring plugin installed but when I click on settings, the box is empty. Is there a way to use an older version of the Ring plugin or can I fill in the JSON manualy? Or any other options.

Thanks
p.s. I'm just started to dip my toe into Homebridge so I'm a newbie atm

**Comment by smockle** [comment, marked as answer]
Support for Node.js versions before 18 was dropped in [`homebridge-ring@12.0.0`](https://github.com/dgreif/ring/releases/tag/homebridge-ring%4012.0.0) (and in [`ring-client-api@12.0.0`](https://github.com/dgreif/ring/releases/tag/ring-client-api%4012.0.0)), so you’ll want to try a version older than 12.0.0. It looks like 11.8.1 is the latest pre-12.0.0 release.

**Comment by tsightler** [comment]
Note that MacOS 10.13.6 is EOL for nearly two years and you will likely have other issues.  For example, ffmpeg-for-homebridge recently upgrade to ffmpeg 6 and, in the process, dropped support for MacOS versions older than 13.0 (Ventura).

You might think this would be no big deal because the package received a major version update, however, @homebridge/camera-utils was updated to include this newer ffmpeg-for-homebridge package even though it only received a minor version bump, so current dependencies pull in @homebridge/camera-utils 2.2.4, which brings ffmpeg-for-homebridge 2.1.1 which no longer supports anything older than MacOS 13.
